index.js

```diff
@@ -3,30 +3,62 @@
  * @param {number[]} rolls - An array of numbers representing rolls on a die.
  * @returns {boolean} Returns `true` if all values in the array are numbers. Otherwise, return `false`.
  */
-function isValid(rolls) {}
+function isValid(rolls) {
+  let number = []
```

Comment:
Here you declared `number` with an empty array but it is never used. What value can the default variable hold and return if the array has all numbers?